app/components/certificate/certificate-display.tsx

```diff
@@ -55,7 +83,7 @@ export default function CertificateDisplay({ title, description, value }: Certif
             }}
             aria-label={`Copy ${title}`}
             icon={<CopyIcon fontSize="md" />}
-            onClick={() => onCopy()}
+            onClick={onCopy}
```

Comment:
This is generally not a great idea, since the call signatures are different.  Here you're passing an `Event` object to your `onCopy` function, which you don't use.  It's often better to use the `() => fn()` form so that you don't implicitly do this.